pkg/storage/filesystem_test.go

```diff
@@ -0,0 +1,32 @@
+package storage
+
+import (
+	"testing"
+)
+
+func TestGetFileSystems(t *testing.T) {
+	fstypes, err :=  internalGetFilesystems("tests/filesystems")
+	expected := [3]string{"ext4", "ext3", "vfat"}
+	if err != nil{
+		t.Error("Failed to open the proc filesystems")
```

Comment:
You can include the error returned by `internalGetFilesystems` here by doing something like `t.Errorf("internalGetFilesystems failed with error %v", err)`
You can do the same thing below for TestEmptyFileSystems too